partner_aging/readme/USAGE.md

```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+1.  Go to Accounting
+2.  Click on Sales \> Customer Aging or Purchases \> Supplier Aging
+3.  Change the date if necessary
```

Comment:
Maybe we could change the path to better match the actual naming? Not detrimental. 

```suggestion
1.  Go to Invoicing
2.  Click on Customers \> Customer Aging or Vendors \> Supplier Aging
3.  Change the date if necessary
```

Reply:
Fixed.